ullyses/timeseries.py

```diff
@@ -366,10 +368,10 @@ def process_files(grating, outfile, indir=".", wavelength_binning=1, min_exptime
     # and to contain the headers used to create the provenance table
     # Ullyses_COSSegmentLists have the Ullyses-specific get_target() and get_coords()
     # methods
-    ensemble = create_ensemble_segmentlist(grating, indir, wavelength_binning, ins=ins)
+    ensemble = create_ensemble_segmentlist(grating, indir, wavelength_binning, ins=ins, infiles=infiles)
     # If there's only 1 matching file for hte specified grating, exit 
     if len(ensemble.datasets) == 1:
-        print(f"SKIPPING: only one matching dataset for grating {grating}")
+        print(f"{RED}SKIPPING: only one matching dataset for grating {grating}{RESET}")
```

Comment:
does this make the line print out red? that's cool!

Reply:
Yes! Some terminal magic